site/content/en/docs/concepts/cluster_queue.md

```diff
@@ -18,23 +18,21 @@ Only [cluster administrators](/docs/tasks#batch-administrator) should create `Cl
 A sample ClusterQueue looks like the following:
 
 ```yaml
-apiVersion: kueue.x-k8s.io/v1alpha2
+apiVersion: kueue.x-k8s.io/v1beta1
 kind: ClusterQueue
 metadata:
-  name: cluster-queue
+  name: "cluster-queue"
```

Comment:
any particular reason to add the quotes? In general it's not needed and I prefer to keep things simple.

Reply:
My two cents:
I lean towards not  putting quotation marks around code unless the quotation marks are mandatory part of the code. I suggest not using quotation consistently.

If the K8s SG says otherwise, please disregard my comment.

Reply:
I couldn't find an official style guide for K8S yamls. I can pass all yamls through some formatter, but would prefer to do it as a separate PR.

Reply:
I also couldn't find a rule about it, but it looks like most samples don't have quotes.

But doing it in a separate PR sounds good